data/databuild.py

```diff
@@ -0,0 +1,18 @@
+import csv
```

Comment:
This script can start with `#!/usr/bin/env python3` so it can be executed automatically

Reply:
I added the hashbang to the script.